### PR TITLE
Add previewOpenDelay props

### PR DESCRIPTION
**Pull request by ahanriat**
Hi 👋 

Just a small PR allowing to add some delay (`previewOpenDelay: number // ms`) for the *row preview*

🚀

### CODE REVIEW

File: SwipeListExample/example.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ import {
 	View
 } from 'react-native';
 
-import { SwipeListView, SwipeRow } from 'react-native-swipe-list-view';
```
Comment (jemise111) [inline SwipeListExample/example.js]: Can you revert this back please?

Comment (ahanriat) [reply SwipeListExample/example.js]: ✅  Sure :)

**Comment by ahanriat** [comment]
@jemise111 this should be good 👌 
Let me know if you need anything else 🙂

**Comment by ahanriat** [comment]
🎉

**Comment by jemise111** [comment]
@ahanriat released with `v1.1.0` thanks again!

**Pull request merged**